README.md

```diff
@@ -14,6 +14,9 @@
         <a href="https://github.com/process-analytics/bpmn-visualization-js/actions">
         <img alt="Build" src="https://github.com/process-analytics/bpmn-visualization-js/workflows/Build/badge.svg"> 
         </a> 
+        <a href="https://sonarcloud.io/dashboard?id=process-analytics_bpmn-visualization-js">
+        <img alt="Coverage" src="https://sonarcloud.io/api/project_badges/measure?project=process-analytics_bpmn-visualization-js&metric=coverage"> 
```

Comment:
:+1: for the badge

![image](https://user-images.githubusercontent.com/27200110/100435268-ff877c00-309d-11eb-8146-a6c2e48904d2.png)